### PR TITLE
add ubuntu 16.04 server

### PR DESCRIPTION
**Pull request by isi-lincoln**
Creates a new makefile target for ubuntu 16.4.3 LTS server.

Ubuntu server is generated using cloud-init, the qcow2 image is 1/5 the size compared to using the ubuntu iso and preseed (~2gb).  There are some minor quirks for ubuntu for renaming interfaces.  This version also uses less scripts than other images, as more data is packaged into the user-data cloud-init script.

Also adds more to the README.md

### CODE REVIEW

**Comment by isi-lincoln** [comment]
I forgot to mention that the main reason the ubuntu image is defaulted to 40gb is that after installing android cruff, 20gb was too small.

File: Makefile [hunk 1]
```diff
@@ -10,14 +10,17 @@
 # =============================================================================
 
 .PHONY: all
-all: fedora debian cumulus freebsd
+all: fedora debian ubuntu cumlulus freebsd
```
Comment (rcgoodfellow) [inline Makefile]: cumlulus?

**Review by rcgoodfellow — CHANGES_REQUESTED** [review]
just fix the typo on 'cumlulus'

File: Makefile (hunk 1 above)
Comment (isi-lincoln) [reply Makefile]: i wondered why git diff highlighted both words... I noticed this... and thought nothing of it.

**Comment by isi-lincoln** [comment]
fixed.

**Review by rcgoodfellow — APPROVED** [review]

**Pull request merged**